common/app/routes/Challenges/views/invalid/NotFound.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+
+// import PropTypes from 'prop-types';
+/* eslint max-len: ["error", { "ignoreStrings": true }] */
```

Comment:
I understand why you need this currently, but long strings and be broken up to be concatenated via javascript

```jsx
<Component
  prop1={ true }
  prop2={ someFn }
  reallyLongStringProp={
    'The first part of the string ' +
    'the second part of the string. ' +
    'and so on'
  }
  />
```

Reply:
Thank you! I fixed the long string and deleted the ignore error eslint comment